src/Questions.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
```

Comment:
Rename Questions.jsx in the QuestionList.jsx, since it is more correctly semantically and intuitively clearer.